### PR TITLE
feat: Setup express boilerplate for GitHub-Label

### LINKED ISSUE #1: Create a project 'Github-Label'
Set up a `Github-Label` application skeleton

- [x] Add folder structure for Backend (Route, Model, Controllers)

- [x] Setup Express, Mongoose

### PR DESCRIPTION
**Pull request by PrateekxCodes**
In this PR I am done with #1  setting up express boilerplate.

### CODE REVIEW

File: feature/Github-Label/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+mongoURL=
```
Comment (harshaanNihal) [inline feature/Github-Label/.env]: Try to keep an empty blank line after every file.

File: feature/Github-Label/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+yarn.lock
+node_modules
```
Comment (harshaanNihal) [inline feature/Github-Label/.gitignore]: Same Here. Empty blank line.

File: feature/Github-Label/app.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const mongoose = require("mongoose");
+
+// Require routes
```
Comment (harshaanNihal) [inline feature/Github-Label/app.js]: These comments don't add any value as code is self-explanatory. 
Unless something is not explicitly clear we don't add comments.

**Review by harshaanNihal — CHANGES_REQUESTED** [review]
Looks good to me.
Added some comments according to BigBinary Coding Guidelines.

File: feature/Github-Label/.env (hunk 1 above)
Comment (PrateekxCodes) [reply feature/Github-Label/.env]: Resolved 👍

File: feature/Github-Label/.gitignore (hunk 2 above)
Comment (PrateekxCodes) [reply feature/Github-Label/.gitignore]: Now fixed 😃

File: feature/Github-Label/app.js (hunk 3 above)
Comment (PrateekxCodes) [reply feature/Github-Label/app.js]: Now its resolved @harshaanNihal

**Comment by harshaanNihal** [comment]
Everything looks fine. You can Merge the PR.

**Pull request merged**